docs/ConvertDataOperation.md

```diff
@@ -0,0 +1,194 @@
+To make a data conversion call, you need a template collection to define the mappings between different data formats.
+
+We have a built-in template collection (hl7v2 to FHIR R4) which is a copy from the FHIR Converter project.
+You can directly use the built-in template collection reference ```microsofthealth/fhirconverter:default``` in the request payload.
+
+If you need to customize the templates, you can get a copy from the FHIR Converter project and customize it as you wish, then upload it to your ACR (Azure Container Registry). In this case, the template collection reference should be ```<RegistryServer>/<imageName>:<imageTag>```. Both image tags and digests are supported in the reference string.
+
+In order to integrate the ACR to your FHIR instance, you need update `FhirServer:Operations:ConvertData:ContainerRegistries:0:RegistryServer` to your ACR login server. Besides, you need to give your App service the ```AcrPull``` permission for the ACR of your choice.
+
+## Request format
+The request payload is passed as [Parameters](http://hl7.org/fhir/parameters.html) resource to the convert data api.
+To convert Hl7v2 data to FHIR, you need to pass four parameters:
+1. *inputData*: raw string content of the input data.
+2. *inputDataType*: data type of your input, currently only accepts *Hl7v2*.
+3. *templateCollectionReference*: reference string of your template collection, can be either the built-in templates ```microsofthealth/fhirconverter:default``` or your custom image reference.
+4. *entryPointTemplate*: the entry point template to process (render) the
+input data.
+
+Here is a sample request to convert data:
+```json
+{
+    "resourceType": "Parameters",
+    "parameter": [
+        {
+            "name": "inputData",
+            "valueString": "MSH|^~\\&|SIMHOSP|SFAC|RAPP|RFAC|20200508131015||ADT^A01|517|T|2.3|||AL||44|ASCII\nEVN|A01|20200508131015|||C005^Whittingham^Sylvia^^^Dr^^^DRNBR^PRSNL^^^ORGDR|\nPID|1|3735064194^^^SIMULATOR MRN^MRN|3735064194^^^SIMULATOR MRN^MRN~2021051528^^^NHSNBR^NHSNMBR||Kinmonth^Joanna^Chelsea^^Ms^^CURRENT||19870624000000|F|||89 Transaction House^Handmaiden Street^Wembley^^FV75 4GJ^GBR^HOME||020 3614 5541^HOME|||||||||C^White - Other^^^||||||||\nPD1|||FAMILY PRACTICE^^12345|\nPV1|1|I|OtherWard^MainRoom^Bed 183^Simulated Hospital^^BED^Main Building^4|28b|||C005^Whittingham^Sylvia^^^Dr^^^DRNBR^PRSNL^^^ORGDR|||CAR|||||||||16094728916771313876^^^^visitid||||||||||||||||||||||ARRIVED|||20200508131015||"
+        },
+        {
+            "name": "templateCollectionReference",
+            "valueString": "microsofthealth/fhirconverter:default"
+        },
+        {
+            "name": "entryPointTemplate",
+            "valueString": "ADT_A01"
+        },
+        {
+            "name": "inputDataType",
+            "valueString": "Hl7V2"
+        }
+    ]
+}
+```
+And the expected output with the built-in template collection:
+```
```

Comment:
nit: would be nice to add `json` after this as in sample request above.

Reply:
Thanks. Have added the `text/plain` content type specification.